package.json

```diff
@@ -289,7 +289,7 @@
     "webpack": "^5.85.0",
     "webpack-bundle-analyzer": "^4.10.1",
     "webpack-cli": "^5.1.1",
-    "webpack-dev-server": "^4.15.0",
+    "webpack-dev-server": "^4.15.2",
```

Comment:
Upgrade `webpack-dev-server` because it uses vulnerable `webpack-dev-middleware` version.